### DISCUSSION TITLE
I can't connect to the Wi-Fi access point.

### DISCUSSION THREAD

**Discussion by ghost** [rern/rAudio#1262] [General]
I installed rAudio-64bit-20230212.img. The Wi-Fi access point is recognized by the android tablet as rAudio, but I cannot connect to the internet. is displayed and connection cannot be established. Please let me know if there is a way to connect. Please excuse any typos. thank you very much.

**Comment by clriis** [comment]
You will need to explain a bit more about your setup. How do you generally connect to the internet? and what do you want to achieve?

**Comment by ghost** [comment]
Thank you for contacting us.
There is something I would like to decline. I am using google translation, so please forgive me if there are any typos.
And I'm new to Linux.
I was able to set the access point with rAudio-1-64bit-20220904, but since the track list is not displayed in the playlist, even if I try to update from the UI, it is from a new image.

When using it outdoors, I would like to connect it self-contained without using a modem device. You need to use an access point for that, follow the contents of rAudio-1/README.md on github, write rAudio-64bit-20230212.img to the SD card, <Edit template file - name and password. Rename wifi0 in BOOT to wifi Edit SSID and Key.>
Then it boots on the Pi3B, but it never connects.
<raudio.local> When connected via wired LAN, it is running but the <wifi> setting is off. In that state, when the wi-fi setting is turned on, rAudio is displayed on the android tablet display, but it keeps trying to connect.
thank you very much.

**Reply by R101G** [reply]
This sounds a bit like the problem here:  #1247
